docs/guides/managing-existing-services.md

```diff
@@ -92,3 +92,62 @@ Harmony validates that the image and tag are reachable - an error will be return
 
 **Important** from the [Docker documentation](https://docs.docker.com/engine/reference/commandline/image_tag/):
 >A tag name may contain lowercase and uppercase characters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.
+
+#### Get the current enable/disable state of the service deployment feature
+
+```
+
+GET {{root}}/service-image-tag/state
+
+```
+**Example {{exampleCounter}}** - Getting the current enable/disable state of the service deployment feature using the `service-image-tag` API
+
+The returned JSON response shows if the service deployment is currently enabled (true) or disabled (false):
+
+```JSON
+{
+  "enabled": true
+}
+```
+---
+**Example {{exampleCounter}}** - Harmony `service-image-tags` response for enable/disable state
+
+#### Enable the service deployment feature
+The user must have admin permission in order to invoke this endpoint.
+
+```
+
+PUT {{root}}/service-image-tag/enable
+
+```
+**Example {{exampleCounter}}** - Enable the service deployment feature using the `service-image-tag` API
+
+The returned JSON response shows if the service deployment is currently enabled:
+
+```JSON
+{
+  "enabled": true
+}
+```
+---
+**Example {{exampleCounter}}** - Harmony `service-image-tags` response for enabling the service deployment
```

Comment:
Who made a guide without supporting {{exampleCounter}}? :) 
This is the result of resolving merge conflict where this is moved from the top level api doc. I will make this hard coded for now. We may want to add a build step to add support of {{exampleCounter}} and {{root}} later.

Reply:
That would not work - these markdown files need to be readable from github, so there is no build step.

Reply:
We can host this page the same way as `app/markdown/api.md` to support {{exampleCounter}} and {{root}}. It doesn't have to render through github.

Reply:
No, it does. The whole point was to not put this on a route since it is not generally available. The AC for the previous ticket was to move it into the github based markdown guides.

Reply:
ok. That makes sense. Didn't know that we were specific about hiding this from the deployed harmony server.